allennlp/fairness/evaluate_bias_mitigation.py

```diff
@@ -0,0 +1,302 @@
+"""
+The `evaluate_bias_mitigation` subcommand can be used to
+compare a bias-mitigated trained model with a baseline
+against an SNLI dataset following the format in [On Measuring
+and Mitigating Biased Inferences of Word Embeddings]
+(https://arxiv.org/pdf/1908.09369.pdf) and reports the
+Net Neutral, Fraction Neutral, and Threshold:tau metrics.
+"""
+
+import argparse
+import json
+import logging
+from typing import Any, Dict, Tuple
+from overrides import overrides
+import tempfile
+import torch
+import os
+
+from allennlp.commands.subcommand import Subcommand
+from allennlp.common import logging as common_logging
+from allennlp.common.util import prepare_environment
+from allennlp.fairness.bias_metrics import NaturalLanguageInference
+from allennlp.data import DataLoader
+from allennlp.models.archival import load_archive
+from allennlp.training.util import evaluate
+
+logger = logging.getLogger(__name__)
+
+
+@Subcommand.register("evaluate-bias-mitigation")
+class EvaluateBiasMitigation(Subcommand):
```

Comment:
We usually define subcommands in `allennlp.commands`. You could just move this file over there.

Reply:
I'm considering just not including this in the core library because it's too specific of a command, @AkshitaB had the great idea to keep it in the guide, so I will just remove it from this PR.

Reply:
@ArjunSubramonian I think you missed this. :)